src/app/components/home/home.component.ts

```diff
@@ -62,17 +79,24 @@ class HomeComponent extends PageComponent implements OnInit {
       .getAuthTrigger()
       .pipe(
         mergeMap(() =>
-          this.projectApi.filter({
+          (settings.hideProjects ? this.regionApi : this.projectApi).filter({
             paging: { items: 3 },
             sorting: { orderBy: "updatedAt", direction: "desc" },
           })
         ),
-        map((data) => List(data.map((project) => project.getCard()))),
+        map((models) =>
+          List(models.map((model: Region | Project) => model.getCard()))
+        ),
         takeUntil(this.unsubscribe)
       )
       .subscribe(
-        (cards) => (this.projectList = cards),
-        () => (this.projectList = List([]))
+        (cards) => {
+          this.viewMore.list = cards;
+          this.viewMore.loading = false;
+        },
+        () => {
+          this.viewMore.loading = false;
```

Comment:
do observables have something equivalent to a `finally` operator?

Reply:
Not in the way you are hoping where we could have a function run when the observable is complete. Observables have the concept of a `complete` operator which is similar to the `finally` operator, but it does not run on failures. The other alternative is the `finalize` pipe, but it would require rewriting tests, and isn't a pretty solution either.